ivadomed/prepare_dataset.py

```diff
@@ -0,0 +1,215 @@
+import nibabel as nib
+import numpy as np
+import matplotlib.pyplot as plt
+import PIL
+import skimage
+import os
+import sys
+from utils import reorient_image
+
+# normalize Image
+def normalize(arr):
+    ma = arr.max()
+    mi = arr.min()
+    return ((arr - mi) / (ma - mi))
+
+
+# Useful function to generate a Gaussian Function on given coordinates. Used to generate groudtruth.
+def label2maskmap_gt(data, shape, c_dx=0, c_dy=0, radius=10, normalize=False):
```

Comment:
maybe it would be more useful for other purposes to have a function that takes as input a n-dim binary array (ones where labels are present), and that would output an array of the same dimension, that would be the convolution of the input array by a Gaussian kernel.  There are probably already a one-liner call to scipy/scikit-learn/numpy that does that.

Reply:
Done in: 839deb4

Reply:
like for the previous function, it would make sense to move it to ivadomed/math. Could you link the issue you are about to open here?

Reply:
issue about the creation of ivadomed/maths.py [here](https://github.com/ivadomed/ivadomed/issues/319)